### PR TITLE
Various Fixes and Quality iof Life changes

### PR DESCRIPTION
**Pull request by rapito**
- Fix SeedBox Download so subtitles are downloaded as well.
https://github.com/popcorn-official/popcorn-desktop/issues/1416
- Place Downloaded files on their own directory, based on the season/episode, defaults to torrent hash if torrent is dropped 
https://github.com/popcorn-official/popcorn-desktop/issues/1408
- Fix Default Settings to set default language to "en", turn off vpn, and automaticUpdating=off
- Fix Subtitles selection when choosing a movie
- Stop showing seedbox view after clicking on Download and show a notification instead so user does not have to go back and look for whatever they were downloading.

### CODE REVIEW

**Comment by team-pct** [comment]
Can one of the admins verify this patch?

**Comment by team-pct** [comment]
Why turn off automaticupdating ?

**Comment by rapito** [comment]
Automatic update often break stable build on people systems. I.e: 
https://www.reddit.com/r/PopCornTime/comments/fk4qqh/previous_version/

It should be off by default and people should choose whether to update or not.

This is not apple or microsoft to force people to do stuff.

**Comment by team-pct** [comment]
I understand that this is not Microsoft but then we will still have people asking why they are not getting updates ...

**Comment by rapito** [comment]
Well... There are no people asking for updates on reddit but they are complaining about builds that break stuff.

There's no need for updates if things are working ok. Whoever needs one would come in and download it.

A better approach for updates is a confirm dialog rather than a mandatory update. Asking the person to update when the app is started if there is a new version. But only update if the user confirms.

**Comment by team-pct** [comment]
Yes on this i agree , updater should be rewritten i will take a look to notify first without installing the updating

File: src/app/lib/providers/opensubtitles.js [hunk 1]
```diff
@@ -47,19 +48,19 @@
         return this.fetch({
             imdbid: id
         }).then(function (data) {
-          App.vent.trigger('update:subtitles', data);
+            App.vent.trigger('update:subtitles', data);
```
Comment (rafaell-lycan) [inline src/app/lib/providers/opensubtitles.js:51]: Could you just turn off you prettier? Otherwise, some small things that shouldn't be committed will be present on your PR.

File: src/app/lib/views/play_control.js [hunk 2]
```diff
@@ -36,12 +36,13 @@
...
       App.vent.on('sub:lang', this.switchSubtitle.bind(this));
       App.vent.on('audio:lang', this.switchAudio.bind(this));
-      App.vent.on(
-        'update:subtitles',
-        function(subs) {
-          this.views.sub.updateLangs(subs);
-        }.bind(this)
-      );
+      App.vent.on('update:subtitles', function(subs) {
+        console.log('update:subtitles');
```
Comment (rafaell-lycan) [inline src/app/lib/views/play_control.js:40]: Can we remove this console?

File: src/app/lib/views/play_control.js [hunk 3]
```diff
@@ -227,16 +215,17 @@
       };
 
       var torrent = providers.torrent.resolveStream
-        ? providers.torrent.resolveStream(
-            defaultTorrent,
-            filters,
-            this.model.attributes
+          ? providers.torrent.resolveStream(
```
Comment (rafaell-lycan) [inline src/app/lib/views/play_control.js:218]: Prettier again.

File: src/app/lib/providers/opensubtitles.js (hunk 1 above)
Comment (rapito) [reply src/app/lib/providers/opensubtitles.js:51]: Agreed, my bad.

File: src/app/lib/views/play_control.js (hunk 2 above)
Comment (rapito) [reply src/app/lib/views/play_control.js:40]: Yup!

File: src/app/lib/views/play_control.js (hunk 3 above)
Comment (rapito) [reply src/app/lib/views/play_control.js:218]: Yeah.... Sorry!

File: src/app/lib/providers/opensubtitles.js (hunk 1 above)
Comment (rapito) [reply src/app/lib/providers/opensubtitles.js:51]: @rafaell-lycan after careful review, I actually made this on purpose becase it was not aligned properly with the rest of the code before, so I made the changes to fix it in here. Take a look how it is now alligned with the rest of the code.

**Comment by test2a** [comment]
What if there is a check for updates before download updates ? That way we can have auto download off by default but check for updates on

**Comment by kiriles90** [comment]
@rapito Is there any chance you could review this? (and maybe leave the controversial parts you were talking about above for a seperate PR so this can be merged?) 

A lot of things changed but some of the issues remain, specifically 

> Fix SeedBox Download so subtitles are downloaded as well. #1416

which now became #1686

> Place Downloaded files on their own directory, based on the season/episode, defaults to torrent hash if torrent is dropped
#1408 

is fixed, we stopped using a hash subdir and instead use the torrent name like before

> Fix Default Settings to set default language to "en", turn off vpn, and automaticUpdating=off

this is the part the maybe leave for a seperate PR until you guys decide what you want to do

> Fix Subtitles selection when choosing a movie

is fixed

> Stop showing seedbox view after clicking on Download and show a notification instead so user does not have to go back and look for whatever they were downloading.

also maybe leave for a seperate PR?

Basically its just #1686, it would be great if you could make a PR with just this so it can be reviewed and merged quickly.
